src/Components/Auth/CustomRegisterForm.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import '../FormStyles.css';
+import { useFormik } from 'formik';
+
+const CustomRegisterForm = () => {
+    const [initialValues, setInitialValues] = useState({
```

Comment:
initial values is a generic name maybe credentials is a more readable name for this view.